app/models/concerns/post.rb

```diff
@@ -0,0 +1,61 @@
+module Post
+  extend ActiveSupport::Concern
+
+  include NameFromTitle
+  include Slug
```

Comment:
Yeah.  I started with just `Name`, but changed it when lots of models implemented name differently.  I'm fine with changing it back, but wanted to make sure it wasn't confusing.